templates/compliance-fields-ar.php

```diff
@@ -10,6 +10,9 @@
 	$document = $order ? get_user_meta( $order->get_user_id(), '_ebanx_document', true ) : false;
 	$address  = $order->get_address();
 
+	$countries_obj = new WC_Countries();
+	$states        = $countries_obj->get_states( 'AR' );
```

Comment:
You could call this variable `$argentina_states`, it's made easy to understand what this variable save when you have to change in the future.